crates/wasmtime/src/runtime/coredump.rs

```diff
@@ -204,6 +204,11 @@ impl WasmCoreDump {
                     Val::AnyRef(_) => {
                         wasm_encoder::ConstExpr::ref_null(wasm_encoder::HeapType::ANY)
                     }
+                    Val::ExnRef(_) => {
+                        todo!(
+                            "Cannot encode a core dump with an exception object global because the wasm_encoder::HeapType does not exist"
+                        )
+                    }
```

Comment:
To avoid panicking during a coredump in the future by accident, could this return an error instead of panicking?

Reply:
One better, it's possible to create an exnref type manually (there's just no alias) -- done.